node/src/BaseClient.ts

```diff
@@ -1056,6 +1057,27 @@ export class BaseClient {
         return this.createWritePromise(createLLen(key));
     }
 
+    /** Sets the list element at index to element.
```

Comment:
```suggestion
    /** 
     * Sets the list element at index to element.
```

Reply:
formatting for all of the other node commands is just like that

Reply:
The file contains formatting both ways. We should align on one moving forward, I created a ticket to align on various node documentation differences. Personally I think its better when it starts on the second line